dotCMS/src/main/java/com/dotcms/storage/ChainableStoragePersistenceAPI.java

```diff
@@ -67,13 +67,14 @@ public boolean existsGroup(final String groupName) {
 
     @Override
     public boolean existsObject(final String groupName, final String objectPath) {
-
         if (this.cache.is404(groupName, objectPath)) {
-
             return false;
         }
-
```

Comment:
if we're this class has a coupled Builder consider making its constructor private or package private

Reply:
Done!